fusor-ember-cli/app/templates/subscriptions/select-subscriptions.hbs

```diff
@@ -4,8 +4,7 @@
       <div class='alert alert-danger rhci-alert'>
           <i class="fa fa-2x fa-exclamation-triangle errorForValidation"></i>
           &nbsp;
-          {{!errorMsg}}
-          Quantity should be greater than zero and should not exceed the number of available subscriptions for this product.
```

Comment:
this is old error message code that never happens since the UI doesn't allow input of great than avail subscriptions.